core/local/watcher.js

```diff
@@ -20,20 +19,27 @@ export interface Watcher {
 }
 */
 
-const log = logger({
-  component: 'LocalWatcher'
-})
-
 function build (syncPath /*: string */, prep /*: Prep */, pouch /*: Pouch */, events /*: EventEmitter */, ignore /*: Ignore */) /*: Watcher */ {
+  let watcher = 'atom'
+  if (process.platform === 'darwin') {
+    watcher = 'chokidar'
+  }
   const env = process.env.COZY_FS_WATCHER
   if (['experimental', 'atom'].includes(env)) {
```

Comment:
Do we really need 2 names for the same thing? Do we really use them?

Also maybe we could make the available watcher types easier to grasp? E.g.:

```js
type WatcherType = 'atom' | 'chokidar'
```